packages/fxa-settings/src/pages/Pair/Supp/WaitForAuth/en.ftl

```diff
@@ -0,0 +1,6 @@
+## WaitForAuth page - Part of the devide pairing flow
+## Users see this page when they have started to pair a second (or more) device to their account
+## The pairing must be approved from both devices to succeed
+
+# The "other device" is non-specific and could be a desktop computer, laptop, tablet, mobile phone, etc.
+pair-wait-for-auth-heading-text = Approval now required <span>from your other device</span>
```

Comment:
```suggestion
# Strings within the <span> elements appear as a subheading.
pair-wait-for-auth-heading-text = Approval now required <span>from your other device</span>
```
Adding a comment from other similar strings to provide context on how this appears.